mlrun/datastore/targets.py

```diff
@@ -930,7 +930,7 @@ def get_spark_options(self, key_column=None, timestamp_key=None, overwrite=True)
                 self.get_target_path()
             )
             storage_spark_options = store.get_spark_options()
-            path = store.url + path
+            path = store.spark_url + path
```

Comment:
Makes no sense. You later pass path to store_path_to_spark(), so this function should build the spark_url basing on the path and the storage_options

Reply:
Resolving duplicate comment.